.github/workflows/inspektor-gadget.yml

```diff
@@ -697,6 +697,48 @@ jobs:
           rm -rf /tmp/.buildx-cache
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
 
+  analyze-and-build:
+    name: IG binary size analysis
+    runs-on: ubuntu-latest
+    needs: 
+      - build-ig 
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@3041bf56c941b39c61721a86cd11f3bb1338122a # v5.2.0
+        with:
+          go-version: '1.23.4'
+          cache: true
+        id: go
+      - name: Install go-size-analyzer
+        run: |
+          go install github.com/Zxilly/go-size-analyzer/cmd/gsa@e47aa2561e70b937f2be7441b2eeca984ff60e8b
+      - name: Get ig-linux-amd64.tar.gz from artifact.
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: ig-linux-amd64-tar-gz
+          path: /home/runner/work/inspektor-gadget/
```

Comment:
Finally, we're are only analyzing the amd64 binary, should we do the same for the arm64 one? (I think it could be done by using a [matrix strategy](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#using-a-matrix-strategy) without adding a lot of lines of code).